### PR TITLE
Compatibility with Rails 5.1

### PR DESCRIPTION
**Pull request by pschambacher**
/cc @grosser @bquorning

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,14 +6,18 @@ rvm:
   - 2.3.4
   - 2.2.7
   - jruby
+bundler_args: ''
```
Comment (grosser) [inline .travis.yml]: this breaks the caching I think since it removes the `--path vendor/bundle` ... do we need it ?

Comment (pschambacher) [inline .travis.yml]: Makes it difficult to run locally because of the `--deployment` argument. It generates a bunch of untracked files for git.

Comment (pschambacher) [inline .travis.yml]: Also it was using `install` before and not `bundler_args`. The first one makes caching impossible when this one is supposed to preserve caching.

**Comment by grosser** [comment]
idk ... might work ... check runtime on travis to see if it uses the cache

**Comment by pschambacher** [comment]
Looks like it does: https://travis-ci.org/zendesk/prop/jobs/230436261#L234

**Comment by grosser** [comment]
Old: bundle install --jobs=3 --retry=3 --deployment
--path=${BUNDLE_PATH:-vendor/bundle}
New: bundle install  --path=${BUNDLE_PATH:-vendor/bundle}

... so should be good

**Pull request merged**